spec/rails_helper.rb

```diff
@@ -62,3 +62,5 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+require 'support/shoulda_matchers'
```

Comment:
You want to add this on line 9 or so just so it's loaded before the config

Reply:
https://github.com/thoughtbot/shoulda-matchers#rails-apps it says "at the bottom of `spec/rails_helper.rb`"

Reply:
Oh I thought it was just the gem import, then you can leave it as it is